### DISCUSSION TITLE
Surfing Key

### DISCUSSION THREAD

**Discussion by lrxyab** [catppuccin/catppuccin#2000] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://github.com/brookhong/Surfingkeys/wiki/Color-Themes
https://github.com/brookhong/Surfingkeys/blob/master/src/content_scripts/ui/frontend.css

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by pufferfish-connoisseur** [comment]
I found [Rose Pine Moon](https://github.com/rose-pine/surfingkeys) to be pretty similar to catppuccin. I have no experience with making a theme myself, but I hope whoever desides to take on this task finds this to be a useful starting point!
